src/routes/view-content/[guideId]/[bibleSection]/context.ts

```diff
@@ -34,6 +34,7 @@ export function createContentContext() {
     const isLoadingToOpenPane = writable(false);
     const isPassageSearch = writable(false);
     const passageSearchBibleSection = writable<BibleSection | null>(null);
+    const fullscreenTextResourceStacksByTab = writable<Map<ContentTabEnum, BasicTextResourceContent[]>>(new Map());
```

Comment:
Moved this into the context since it was getting prop-drilled through with `bind:...`. Much cleaner this way and we can export the `pushToFullscreenTextResourceStack` to simplify most usages.